lib/src/editor/ui/editor.dart

```diff
@@ -60,9 +60,10 @@ class Editor extends StatelessWidget {
                     ),
                     Consumer<ZoomProvider>(
                       builder: (context, model, child) {
-                        return Transform.scale(
-                          scale: model.zoom,
-                          child: component ?? const SizedBox.shrink(),
+                        return DevicePreview(
```

Comment:
Does `DevicePreview` have it's own zoom controls on the new panel? 

If so, we should remove the existing zoom controls, or wire them up to control the zoom controls inside the  `DevicePreview`.

If `DevicePreview` does not have zoom controls, then you should try to add back the `Transform.scale` inside the `builder` below